stylize.py

```diff
@@ -1,11 +1,13 @@
 # Copyright (c) 2015-2018 Anish Athalye. Released under GPLv3.
 
+import os
 import vgg
 
 import tensorflow as tf
+# https://stackoverflow.com/a/42121886
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '2'
```

Comment:
Could you move this bit into `main()` inside neural_style.py?

If someone wants to e.g. use stylize.py as a library, we shouldn't set environment variables for them.

Reply:
Good point. Now that I think of it, I should also check if it is already set to let people override the default.